package/kernel/linux/modules/ipmi.mk

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright (C) 2022 TDT AG <development@tdt.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See https://www.gnu.org/licenses/gpl-2.0.txt for more information
+#
+
+IPMI_MENU:=IPMI system interface support
+
+define KernelPackage/ipmi-handler
+  SUBMENU:=$(IPMI_MENU)
+  TITLE:=IPMI top-level message handler
+  KCONFIG:=CONFIG_IPMI_HANDLER=m \
+           CONFIG_ACPI_IPMI=m \
+           CONFIG_IPMI_PANIC_EVENT=y \
+           CONFIG_IPMI_PANIC_STRING=y \
+           CONFIG_IPMI_DEVICE_INTERFACE=n \
+           CONFIG_IPMI_SI=n \
+           CONFIG_IPMI_SSIF=n \
+           CONFIG_IPMI_WATCHDOG=n \
+           CONFIG_IPMI_POWEROFF=n \
+           CONFIG_SENSORS_IBMAEM=n \
+           CONFIG_SENSORS_IBMPEX=n
+  FILES:=$(LINUX_DIR)/drivers/char/ipmi/ipmi_msghandler.ko
```

Comment:
Heh, this threw me for a loop. But: drivers/char/ipmi/Makefile states:

     obj-$(CONFIG_IPMI_HANDLER) += ipmi_msghandler.o

so, in case someone else stumbles over this and gets confused.

Reply:
So, this is only a hint without anything to do, isn't it?